controllers/tab/workflow/PKPWorkflowTabHandler.inc.php

```diff
@@ -131,13 +131,58 @@ function fetchTab($args, $request) {
 				$templateMgr->assign('editingNotificationRequestOptions', $notificationRequestOptions);
 				return $templateMgr->fetchJson('controllers/tab/workflow/editorial.tpl');
 			case WORKFLOW_STAGE_ID_PRODUCTION:
+				$selectedStageId = $this->getAuthorizedContextObject(ASSOC_TYPE_WORKFLOW_STAGE);
+
 				$templateMgr = TemplateManager::getManager($request);
 				$notificationRequestOptions = $this->getProductionNotificationOptions($submission->getId());
 				$representationDao = Application::getRepresentationDAO();
 				$representations = $representationDao->getBySubmissionId($submission->getId());
 				$templateMgr->assign('representations', $representations->toAssociativeArray());
 
 				$templateMgr->assign('productionNotificationRequestOptions', $notificationRequestOptions);
+
+				// Versioning
+				$context = $request->getContext();
+				$templateMgr->assign('versioningEnabled', $context->getSetting('versioningEnabled'));
+
+				// get all revisions of this submission
+				$articleDao = DAORegistry::getDAO('ArticleDAO');
```

Comment:
application specific classes (articleDao) should/can not be used in pkp-lib, but Application::getSubmissionDao() or so...